browser_patches/firefox/preferences/playwright.cfg

```diff
@@ -14,6 +14,12 @@ pref("browser.tabs.remote.useCrossOriginOpenerPolicy", false);
 
 pref("browser.tabs.remote.separatePrivilegedMozillaWebContentProcess", false);
 
+pref("pdfjs.disabled", true);
```

Comment:
I'm curious, why did do we disable it?

Reply:
This way navigating to PDF triggers download instead of opening the inline plugin.

Reply:
sure, but did it broke tests or are we just randomly adding it?

Reply:
it did!